src/pg/sql/helper.js

```diff
@@ -0,0 +1,47 @@
+import sql, { raw, join } from 'sql-template-tag';
+
+export const nowTimestamp = sql`cast(extract(epoch from now()) as integer)`;
+
+export const getJsonBuildObject = (variadic) => {
+  const args = join(
+    Object.entries(variadic).map(([name, value]) => {
+      return sql`'${raw(name)}', ${value}`;
+    }),
+  );
+  return sql`jsonb_build_object(${args})`;
+};
+
+export const getSelectCols = (table) => {
+  // TODO: When we have a query object, get only the requested columns.
+  return sql`to_jsonb("${raw(table)}")`;
+};
+
+export const colsAndValues = (data, versionCol) => ({
+  cols: join(
+    Object.keys(data)
+      .map((col) => raw(`"${col}"`))
+      .concat(raw(`"${versionCol}"`)),
+    ', ',
+  ),
+  values: join(Object.values(data).concat(nowTimestamp), ', '),
+});
+
+export const whereConditions = (data) =>
+  join(
+    Object.entries(data).map(([name, value]) => {
+      return sql`"${raw(name)}" = ${value}`;
+    }),
+  );
+
+export const getUpdates = (data, options) => {
+  const { id, version } = options;
+  return join(
+    Object.entries(data)
+      .filter(([name]) => name !== id)
+      .map(([name, value]) => {
+        return sql`"${raw(name)}" = ${value}`;
+      })
+      .concat(sql`"${raw(version)}" =  ${nowTimestamp}`),
+    ', ',
+  );
+};
```

Comment:
I think we can delete these 3 unused functions, there is more complete implemetations in getArgSql.js and upsert.js